jetty-core/jetty-server/src/main/config/modules/dos.mod

```diff
@@ -0,0 +1,56 @@
+# DO NOT EDIT THIS FILE - See: https://eclipse.dev/jetty/documentation/
+
+[description]
+Enables the DosHandler for the server.
+
+[tags]
+connector
+
+[depend]
+server
+
+[xml]
+etc/jetty-dos.xml
+
+[ini-template]
+
+## The algorithm to use for obtaining an Id from an Request: ID_FROM_REMOTE_ADDRESS, ID_FROM_REMOTE_PORT, ID_FROM_REMOTE_ADDRESS_PORT, ID_CONNECTION
+#jetty.dos.id.type=ID_FROM_REMOTE_ADDRESS
+#jetty.dos.id.class=org.eclipse.jetty.server.handler.DosHandler
+
+## The class to use to create Tracker instances to track the rate of requests
+#jetty.dos.rateControlFactory=org.eclipse.jetty.server.handler.DoSHandler$LeakingBucketTrackerFactory
+
+## The sample period(ms) to determine the request rate, or -1 for a default value
+#jetty.dos.trackerFactory.samplePeriodMs=1000
+
+## The maximum requests per second per client
+#jetty.dos.trackerFactory.maxRequestsPerSecond=100
```

Comment:
Is this per second or per sample period?

Reply:
As it says "PerSecond"